projects/RabbitMQ.Client.OpenTelemetry/RabbitMQ.Client.OpenTelemetry.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@WeihanLi whatever text editor you are using is removing the UTF-8 BOM from these files, and creating an unnecessary change. Please revert.

Reply:
@lukebakken  thanks very much for help, I did not notice this before